contributing.md

```diff
@@ -0,0 +1,3 @@
+[![Jazzband](https://jazzband.co/static/img/jazzband.svg)](https://jazzband.co/)
```

Comment:
@Archmonger Let's upper-case the filename please.

Reply:
https://jazzband.co/static/img/Jazzband.svg and JAZZBAND.svg is are 404. Perhaps I'm misunderstanding?